client/packages/requisitions/src/RnRForms/api/hooks/useRnRFormContext.ts

```diff
@@ -1,41 +1,28 @@
-import { RnRFormNodeStatus } from '@common/types';
 import { RnRFormLineFragment } from '../operations.generated';
-import { ArrayUtils, create } from '@openmsupply-client/common';
-
-export interface RnRForm {
-  id: string;
-  periodLength: number;
-  periodName: string;
-  status: RnRFormNodeStatus;
-  lineIds: string[];
-}
-
-export interface RnRFormLine extends RnRFormLineFragment {
-  isDirty?: boolean;
-}
+import { create } from '@openmsupply-client/common';
 
 interface RnRFormContext {
-  isLoading: boolean;
-  form: RnRForm | undefined;
-  lines: Record<string, RnRFormLine>;
-  setForm: (form: RnRForm) => void;
-  setIsLoading: (isLoading: boolean) => void;
-  setLine: (line: RnRFormLine) => void;
-  setLines: (lines: RnRFormLineFragment[]) => void;
+  dirtyLines: Record<string, RnRFormLineFragment>;
```

Comment:
:nit the inconsistent naming is bothering me.
can you change 'dirty' to 'draft' plz? We have `setDraftLine` which updates the `dirtyLines`